app/templates/briefs/_brief_response_data.html

```diff
@@ -0,0 +1,29 @@
+{% import "toolkit/summary-table.html" as summary %}
+{% for section in content.summary(brief_response) %}
+    {% if not (section.id == 'your-nice-to-have-skills-and-experience' and not section.get_question('niceToHaveRequirements').boolean_list_questions) %}
```

Comment:
I know it's ugly but I couldn't find any other good way round it... it's tricky because it's not an empty section - it still has the `niceToHave` question, but the question happens to not have any `boolean_list_questions` set in it.